app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    files = command.split(" ")
```

Comment:
unpacke the command.split(" ") into relevant variables, making it easier to understand what the command contains.

as well check if the command was correct and consists of three parts